src/components/SideBarComponents/SideBar.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+import { useState } from "react";
+import styles from "./SideBar.module.css";
+import SideBarItem from "./SideBarItem";
+import SideSearch from "./SideSearch";
+
+function SideBar() {
+  const data = [
```

Comment:
move const  to utils -> consts -> sidebar.consts.ts

Reply:
?

Reply:
g